src/detections/table/header/items.js

```diff
@@ -0,0 +1,7 @@
+export const titles = [
+  { label: 'ID', hideOnMobile: true },
+  { label: 'CLASS', hideOnMobile: false },
```

Comment:
For the headers that shouldn't be hidden on mobile, we can remove the `hideOnMobile` field entirely, as trying to get it into our hook would then return `undefined`, a falsly value anyways.

Reply:
Done